src/archana/SeleniumAss1.java

```diff
@@ -0,0 +1,67 @@
+package archana;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class SeleniumAss1 {
+	static WebDriver driver;
+
+	void launchBrowser() {
+		// Launch chrome with form
+		System.setProperty("webdriver.chrome.driver", "E:\\JAVA_class\\chromedriver.exe");
+		driver = new ChromeDriver();
+		driver.get("file:///E:/JAVA_class/Selenium/Ass1/SeleniumAssignment_1.html");
+		// verify Title
+		String title = driver.getTitle();
+		if (title.equals("TECHNOCREDITS"))
+			System.out.println("Correct page loaded");
+		else
+			System.out.println("Incorrect page loaded");
+	}
+
+	void fillAllDetails() {
+		driver.findElement(By.id("first name")).sendKeys("Archana");
+		driver.findElement(By.id("last name")).sendKeys("Nawale");
+		driver.findElement(By.id("E-mail")).sendKeys("A.N@gmail.com");
+		driver.findElement(By.id("Company Name")).sendKeys("Techno");
+		if (driver.findElement(By.id("maleG")).isSelected())
```

Comment:
You Swap the validation, for Gender radio button if ‘Female’ radio button is by default selected then select ‘Male’   radio button and vice versa.